File: add-suggestions-state.py
#!/usr/bin/env python3
import sys

# Read the file
with open('frontend/src/pages/StrategicDashboard.jsx', 'r') as f:
    lines = f.readlines()

# Find line 574 (0-indexed: 573) and insert after it
insert_at = 574  # Line number to insert after (1-indexed)
lines.insert(insert_at, '  const [suggestionsData, setSuggestionsData] = useState(null);\n')

# Write back
with open('frontend/src/pages/StrategicDashboard.jsx', 'w') as f:
    f.writelines(lines)

print(f"Added suggestions state at line {insert_at + 1}")

File: add-suggestions-state.mjs
import fs from 'fs';

// Read the file
const content = fs.readFileSync('frontend/src/pages/StrategicDashboard.jsx', 'utf8');
const lines = content.split('\n');

// Find line with competitivenessData and insert suggestionsData after it
const insertIndex = lines.findIndex(line =>
  line.includes('const [competitivenessData, setCompetitivenessData] = useState(null)')
);

if (insertIndex !== -1) {
  lines.splice(insertIndex + 1, 0, '  const [suggestionsData, setSuggestionsData] = useState(null);');
  fs.writeFileSync('frontend/src/pages/StrategicDashboard.jsx', lines.join('\n'));
  console.log(`Added suggestions state at line ${insertIndex + 2}`);
} else {
  console.log('Could not find competitivenessData line');
}
